build/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM registry.access.redhat.com/ubi7/ubi
```

Comment:
could you please use a minimal image here?

Reply:
minimal image does not contain yum - it's kind of pain due since Microdnf does not work with multiple gpgkey in repo file.
Also minimal image by default can't provide python package required for gcloud package

Reply:
multiple gpg keys are required for google yum repo

Reply:
done